roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -163,6 +180,64 @@
     "get_lsn": "rule:admin_only",
     "create_lsn": "rule:admin_only",
 
+    "create_flavor": "rule:admin_only",
```

Comment:
We should let the neutron SMEs take a look at all the new apis and tell us the access control.